js/iframe-coordinator/tslint.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": ["tslint:recommended", "tslint-config-prettier"],
+    "linterOptions": {
+      "exclude": ["node_modules/**/*.ts"]
+    },
+    "rules": {
```

Comment:
There are a few differences to the rules that I wanted to point out:
- member-access dropped no-public.  This was because Justin had originally requested u add public.  I am fine with either if we want to match.
- member ordering was added.  I like grouping related methods rather than forcing up to have public than private.
- no-trailing-whitespace was added.
- object-literal-sort-keys was added.  Having to alphabetically order keys in objects is annoying to me.
- variable-name was added.  We were using _ for private members

Reply:
I would like object-literal-sort-keys if there was a way to autoformat it.  I'd have opted for enabling member ordering, particularly since it has a fixer which means it can probably be automated via editor config. That said, it's not a big deal and there are better ways for us to spend time than bike-shedding linter options. :grin: